src/plot.jl

```diff
@@ -86,14 +88,6 @@ struct Fan; x; y; tipannotations; marker_x; marker_y; showtips; tipfont; marker_
             end
         end
     end
-    if dend.showtips
-        @series begin
-            seriestype := :scatter
-            markersize := 0
-            series_annotations := map(x -> text(x[3], :left, dend.tipfont...), dend.tipannotations)
-            getindex.(dend.tipannotations, 1), getindex.(dend.tipannotations, 2)
-        end
-    end
     primary = false
     label = ""
```

Comment:
also what are these two lines doing? are they meant to be ``primary := false; label := ""``? or just deleted?

Reply:
`primary := false; label := ""` as you say :+1:

Reply:
do you mind updating that before I merge?